package.json

```diff
@@ -139,7 +139,8 @@
     "coverageReporters": [
       "json",
       "json-summary",
-      "text"
+      "text",
+      "clover"
```

Comment:
TIL about clover! very cool. from this config it looks like collecting JS test coverage metrics isn't too difficult! definitely something we'd love to get your guidance with on the product side, I think, at some point :)

Reply:
Right?! I think it should be pretty easy. I believe I just need to install `codecov` then run it in CI and it _should_ upload the results for us.

Would be more than happy to help on the product side :D